ga4gh/datamodel/variants.py

```diff
@@ -48,6 +48,13 @@ def convertVCFGenotype(vcfGenotype, vcfPhaseset):
     return genotype, phaseset
 
 
+def variantSetFactory(variantSetId, relativePath):
+    if True:  # TODO look for '.wt' in some part of directory path
```

Comment:
@jeromekelleher can you provide some clarity here?  (see question on issue page)